packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -224,8 +223,8 @@ class TimeInput extends React.Component {
     // eslint-disable-next-line react/no-did-update-set-state
     this.setState({
       hour,
-      minute: TimeUtil.splitMinute(this.props.value),
-      second: TimeUtil.splitSecond(this.props.value),
+      minute,
+      second,
```

Comment:
Looks like these changes are just code formatting and moving the declarations up. Do you want to keep these changes or get rid of them? If keeping them, I suggest adding an actual entry in the change log.

Reply:
Thank you I missed it, I would keep Terra-Time-Input to be unchanged for this Issue. since Missing hour will handled by date-time-picker. : 4aaa072145241a931da717a591d0a5f9e644356f